### PR TITLE
stubtest: fix on windows

### LINKED ISSUE #4448: stubtest on windows isn't doing anything
I think this has been the case for a while, I suspect since they were moved to Github actions in #4223 
Example in https://github.com/python/typeshed/pull/4447

### PR DESCRIPTION
**Pull request by hauntsaninja**
Resolves #4448

Co-authored by: Akuli

### CODE REVIEW

File: stdlib/2and3/time.pyi [hunk 1]
```diff
@@ -69,10 +69,11 @@ if sys.version_info >= (3, 3):
             ],
             _arg: Any = ...,
         ) -> struct_time: ...
-        @property
-        def tm_zone(self) -> str: ...
-        @property
-        def tm_gmtoff(self) -> int: ...
+        if sys.version_info >= (3, 6):
+            @property
+            def tm_zone(self) -> str: ...
```
Comment (JelleZijlstra) [inline stdlib/2and3/time.pyi:74]: This does exist in 3.5 on Linux.

Comment (Akuli) [reply stdlib/2and3/time.pyi:74]: The docs say that it's platform specific before Python 3.6 https://docs.python.org/3/library/time.html

**Comment by Akuli** [comment]
There might be non-windows systems that lack `tm_zone`, and those will type check pass and get runtime error. I guess that's preferred over failing type check but working correctly at runtime?

**Comment by hauntsaninja** [comment]
Yup! Also generally speaking, we don't do a great job of platforms outside of linux, win32, darwin (and up until we got stubtest running on windows, win32 had a lot of room for improvement too). For instance, we run flake8-pyi in typeshed CI and it currently only recognises four sys.platform values: https://github.com/ambv/flake8-pyi/blob/cdd527b601d95da1b5223a6be54ce0c675771b4b/pyi.py#L271 And of course, we'd be receptive to improvements for other platforms if we ever got issues or PRs :-)

**Pull request merged**